[WIP] Update documentation for Solver Setup (#81)
su2code/su2code.github.io: _docs_v7/Solver-Setup.md

This is a basic introduction on how to set up a simulation using SU2. We distinguish between single-zone computations and multi-zone computations. The following considers a single zone only. For an explanation on multi-zone problems, continue with [Basics of Multi-Zone Computations](/docs_v7/Multizone).

> Three different types of mathematical problems can be solved in SU2. The type of problem to be solved is specified on the config file by the `MATH_PROBLEM` field. The options are:

Reviewer: ```suggestion
Three different types of mathematical problem can be solved in SU2. The type is specified via the `MATH_PROBLEM` config option. The options are:
```

`DIRECT`: Also referred to as primal, this is the default when SU2_CFD or SU2_DEF are used.

> `DISCRETE_ADJOINT`: A discrete adjoint methodology based on Automatic Differentiation, available for most solvers.

Reviewer: ```suggestion
`DISCRETE_ADJOINT`: A discrete adjoint methodology based on Automatic Differentiation, available for most solvers. This is the default for SU2_CFD_AD and SU2_DOT_AD
```

 - `NONE`: No turbulence model.
 - `SST`: Mentor Shear Stress Transport.
 - `SA`: Spalart-Allmaras.
 
Different corrections and variations are implemented for each turbulence model. These are specified accordingly either in `SST_OPTIONS` or in `SA_OPTIONS`. The folowing options are available:
- `SST_OPTIONS`:
  - `NONE`: No SST turbulence model (default).
  - `V1994m`: refers to the "Standard" Menter SST model from 1994 with the modified constant $\sigma_{k1}$ and the redefinition of the turbulent eddy viscosity.
  - `V2003m`: refers to the Menter SST model from 2003 ignoring the $\tau_{ij}$ in the momentum and energy equations and the production term is approximated by $P = \mu_t S^2$.
  - `VORTICITY`: refers to the vorticity source term correction.
  - `KATO_LAUNDER`: refers to the Kato-Launder correction.
  - `UQ`: refers to the V1994m version with uncertainty quantification modifications.??
  - `SUSTAINING`: refers to the controlled decay correction.

- `SA_OPTIONS`:

>   - `NONE`: refers to the modification where the $f_{t2}$ term is set to zero, i.e., $c_{t3} = 0$ (default).

Reviewer: This does not have to be repeated here... just point to the section in Physical definition.

#### Spalart-Allamaras ####
The single transported Spalart-Allmaras variable $\tilde{\nu}$ is initialized with the value at the farfield or inlet boundary. As suggested in the literature, the value there is computed as $\tilde{\nu}/\nu=\text{turb2lam}$. In SU2 the free-stream Spalart-Allmaras variable to kinematic laminar viscosity ratio, $\text{turb2lam}$, is defined by the `FREESTREAM_NU_FACTOR` config file field. The default value is $\tilde{\nu}/\nu = 3.0$ avoiding laminar solutions.

An extension of SU2 includes an hybrid turbulence model: the Spalart-Allmaras original model with Detached-Eddy Simulation (DES) modification. Refer to Eduardo Moina's thesis?. The use of the hybrid RANS/LES model is specified in the cofig file with the field `HYBRID_RANSLES`. Four different techniques are currently implemented:

- `SA_DES` Detached-Eddy Simulation
- `SA_DDES` Delayed Detached-Eddy Simulation
- `SA_ZDES` Zonal Detached-Eddy Simulation
- `SA_EDDES` Enhanced Detached-Eddy Simulation

The DES constant can be controlledd by the field `DES_CONST`, with 0.65 as default.

#### Menter’s k-omega SST Model ####

> As initial conditions, the values of are initialized at all grid point with the farfield values. The farfield conditions for the turbulent kinetic energy $k$ is computed as: $k = \frac{3}{2}(UI)$, whereby $U$ is the freestream velocity magnitude and $I$ is the turbulence intensity. This latter value can be specified in the config file by the `FREESTREAM_TURBULENCEINTENSITY` field, with defaul value to 0.05 which corresponds to a $5\%$. The freestream value of the turbulent frequency $\omega$ is computed as: $\omega = \frac{\rho k}{\mu}(\frac{\mu_t}{\mu})^{-1}$, where $\rho$ and $\mu$ are the freestream density and laminar viscosity respectively and $\frac{\mu_t}{\mu}$ is the turbulent viscosity ratio, also known as freestream dissipation. Its value can be specified in the `FREESTREAM_TURB2LAMVISCRATIO` field, defaulting to $10$.

Author: Would be nice if someone checks the boundary conditions for omega. I am not that familiar with the k-w model.
